src/Parsers/ASTFunction.cpp

```diff
@@ -622,51 +619,17 @@ ASTPtr ASTFunction::toLiteral() const
 }
 
 
-/** A special hack. If it's [I]LIKE or NOT [I]LIKE expression and the right hand side is a string literal,
-  *  we will highlight unescaped metacharacters % and _ in string literal for convenience.
-  * Motivation: most people are unaware that _ is a metacharacter and forgot to properly escape it with two backslashes.
-  * With highlighting we make it clearly obvious.
-  *
-  * Another case is regexp match. Suppose the user types match(URL, 'www.clickhouse.com'). It often means that the user is unaware that . is a metacharacter.
-  */
-static bool highlightStringLiteralWithMetacharacters(const ASTPtr & node, const IAST::FormatSettings & settings, const char * metacharacters)
```

Comment:
Why did `highlightStringLiteralWithMetacharacters` disappear?

Reply:
Moved to `FormattingBuffer` because it's called when `hilite` flag is set, but I've encapsulated the hilite flag, and don't want to expose a getter for just this one usage.